gangcaiyoule/app.py

```
@@ -0,0 +1,286 @@
+
+def load_conversation_history(conversation_id):
+    url = f"http://localhost:8080/history/{conversation_id}"
+    response = session.get(url)
+    if response.status_code == 200:
+        history = response.json().get('history', [])
+        chat = []
+        current_user_msg = None
+
+        for h in history:
+            if h['role'] == 'user':
+                # 遇到用户消息，暂存
```

Ruff (0.13.3)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)